src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -423,7 +439,8 @@ namespace AppInstaller::CLI::Workflow
     {
         context <<
             Workflow::DownloadSinglePackage <<
-            Workflow::InstallPackageInstaller;
+            Workflow::InstallPackageInstaller <<
+            Workflow::DisplayInstallationNotes;

```

Comment:
Why not make it part of `InstallPackageInstaller`?

Reply:
Moved to `InstallPackageInstaller`